rocksdb/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+mod r#move;
```

Comment:
Maybe we can call this `persistence` or just `persist` as in the storage layer.

Reply:
I agree a different name is needed here. I don't like using reserved words even if we can do it via the `r#` trick.